core/jobs.py

```diff
@@ -205,13 +205,13 @@ def register_completion(cls, job_id, output_list):
 
         model.status_code = STATUS_CODE_COMPLETED
         model.time_finished_msec = utils.get_current_time_in_millisecs()
-        model.output = cls._compress_output_list(output_list)
+        model.output = cls.compress_output_list(output_list)
```

Comment:
See my comment below.

Reply:
I have modified it back to private function